.github/workflows/test-origami-component.yml

```diff
@@ -9,4 +9,10 @@ jobs:
       with:
         node-version: 12.x
     - run: npm install --only=dev
-    - run: npx origami-ci branch
\ No newline at end of file
+    - name: Repository name
+      id: repo_name
+      run: echo ::set-output name=REPO_NAME::$(echo $GITHUB_REPOSITORY | cut -d / -f 2)
```

Comment:
what shell are we in here? if it's bash we can do `name=REPO_NAME::${GITHUB_REPOSITORY/*\//}`

Reply:
Let's try it
```suggestion
      run: echo ::set-output name=REPO_NAME::${GITHUB_REPOSITORY/*\//}
```